decidim-accountability/app/permissions/decidim/accountability/admin/permissions.rb

```diff
@@ -32,13 +33,23 @@ def can_perform_actions_on?(subject, resource)
           return unless permission_action.subject == subject
           return false if can_create_grandchildren_results?
 
-          case permission_action.action
-          when :create, :create_children
-            true
-          when :update, :destroy
-            resource.present?
+          case permission_action.subject
+          when :import_projects
+            case permission_action.action
+            when :create, :new
+              true
```

Comment:
I just realized that we have a problem here in case the `budgets` component is not available.

This permission should only return true in case `Decidim::Budgets::Project` is defined i.e. `defined?(Decidim::Budgets::Project)`.

I believe the same bug actually exists at budgets component (import proposals) but let's not re-create the bug here.

Otherwise the import views would be broken in case the `budgets` component is not available.